generate/run_ioi_slurm.py

```diff
@@ -9,7 +9,7 @@
 
 logger = logging.getLogger(__name__)
 
-DEFAULT_TP = 16
+DEFAULT_TP = 8
```

Comment:
If I'm not mistaken, one cannot have TP > 8 on a single node.

Reply:
if you see further down there's this line:
```
n_nodes = ceil(tp / 8)
```

Reply:
I see, although why set the default to 16? Were you thinking about multi-node inference?